packages/opentelemetry-test-utils/testUtils.ts

```diff
@@ -126,3 +126,15 @@ export const assertPropagation = (
   );
   assert.notStrictEqual(targetSpanContext.spanId, sourceSpanContext.spanId);
 };
+
+/**
+ * Represents a timed event.
+ * A timed event is an event with a timestamp.
+ */
+export interface TimedEvent {
```

Comment:
should be able to use the type from the `tracing` module here

Reply:
the tracing doesn't have this interface yet

Reply:
I think it should be in tracing since it's a tracing sdk specific thing

Reply:
if this is just temporary you can just put a comment to remove later

Reply:
but it doesn't exist there yet, besides it is just one interface needed for test utils, we can change it later or leave it as it is. I'm fine for both, but for now we cant use it.

Reply:
ok